client/src/pages/CreatePokemon.jsx

```diff
@@ -0,0 +1,150 @@
+                            hideInterface={true}
+                            brushColor={color}
+                            brushRadius={5}
+                            lazyRadius={0}
+                            ref={canvasInput}
+                            style={{ width: '100%', height: '100%' }} />
+                    </div>
+                    <div className="canvas-options">
+                        <div className="color-picker">
+                            <label htmlFor="select-color">Color</label>
+                            <input type="color" id="select-color" value={color} onChange={(e) => setColor(e.target.value)}></input>
+                        </div>
+                        <button onClick={(e) => {
+                            e.preventDefault();
+                            canvasInput.current.clear();
+                        }}>Clear</button>
+                        <button onClick={(e) => {
+                            e.preventDefault();
+                            canvasInput.current.undo();
+                        }}>Undo</button>
+                    </div>
+                </div>
+                <div className="pokemon-form">
+                    <h2>Create a Pokemon</h2>
+                    <div className="name-row">
+                        <label htmlFor="name">Name</label>
+                        <input type="text" id="name" name="name" defaultValue={name} required></input>
+                        <button type="button" onClick={(e) => {
+                            e.preventDefault();
+                            randName();
+                        }}>Generate Random Name</button>
+                    </div>
+                    <div className="desc-row">
+                        <label htmlFor="desc">Description</label>
+                        <input type="text" id="desc" name="desc" required></input>
+                    </div>
+                    <div className="type-row">
+                        <label htmlFor="type1">Type 1</label>
+                        <Type id={'type1'} />
+                        <label htmlFor="type2">Type 2</label>
+                        <Type id={'type2'} />
+                    </div>
+                    <div className="move-row">
+                        <label htmlFor="move1">Move 1</label>
+                        <input type="text" id="move1" name="move1" required></input>
+                        <label htmlFor="move1type">Type</label>
+                        <Type id={'move1type'} />
+                        <label htmlFor="move1power">Power</label>
+                        <input type="number" id="move1power" name="move1power" min="0" max="100" defaultValue="0"></input>
+                    </div>
+                    <div className="move-row">
+                        <label htmlFor="move2">Move 2</label>
+                        <input type="text" id="move2" name="move2" required></input>
+                        <label htmlFor="move2type">Type</label>
```

Comment:
I'm not sure this needs to be required - pokemon can have less than 4 moves so I think we should make only move 1 required and the rest optional